packages/react/src/patterns/blocks/FeatureCard/README.md

```diff
@@ -0,0 +1,94 @@
+# FeatureCard
```

Comment:
English name should have a space I believe. In all honesty I'm not sure we're catching all of these, we might need to do a sweep later to align it all again.